### ISSUE TITLE
Unable to remove ASD manager

### ISSUE THREAD

**Issue by simonOpenV** [openvstorage/alba-asdmanager#313]
### Problem description
```
root@kvh195-5:~# asd-manager remove

+++++++++++++++++++++++++++++
+++  ASD Manager removal  +++
+++++++++++++++++++++++++++++
 - Validating configuration management
  - Retrieving ASD information
Are you sure you want to continue? (y/[n]): y
 - Removing from configuration management
 - Removing services
    - Removing service alba-maintenance_mybackend-8D9raTYEiwlMbXIj
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "source/asdmanager.py", line 239, in remove
    API.remove_maintenance_service.original(name=service_name, alba_backend_guid=guid)
TypeError: remove_maintenance_service() got an unexpected keyword argument 'alba_backend_guid'
```

**Comment by JeffreyDevloo** [comment]
Please branch off from Master. The recent promote might have caused this issue

**Comment by simonOpenV** [comment]
Will be fixed by #316

**Comment by JeffreyDevloo** [comment]
Released in https://github.com/openvstorage/alba-asdmanager/releases/tag/1.12.0
Packaged in openvstorage-sdm_1.12.0-1_amd64.deb
